specs/pvs/corla.pvs

```diff
@@ -132,14 +132,21 @@
 fafessl: THEORY
```

Comment:
Hmm - looking again (at my cleaned up version of the example CVR that github can show in table form: https://github.com/nealmcb/ColoradoRLA/blob/cleancvr/test/dominion-2017-CVR_Export_20170310104116-clean.csv) I don't see what I thought I saw last week. Might have confused it with the previous field (PrecinctPortion). But the file is so abbreviated it's hard to know. Anyway if they say its a 1-1 mapping, that's what we want. Thanks, and sorry for the confusion. Indexing on ImprintedId would seem to get things out of order if it is a string and we want stuff in order by numeric tabulator num etc.